modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -78,7 +88,7 @@ The channels you need for this procedure are:
 
 |===
 
-include::snippets/add_channels_cli.adoc[]
+include::snippets/add_channels_scc.adoc[]
```

Comment:
No SCC for Uyuni, only SUSE Manager.

Reply:
![image](https://user-images.githubusercontent.com/3914967/98321761-9af27780-2031-11eb-9161-919c9a638894.png)

Reply:
Same story. Using the name SCC to refer to `spacewalk-common-channels` generates confusion.